es-app/src/guis/GuiCollectionSystemsOptions.cpp

```diff
@@ -138,6 +142,15 @@ void GuiCollectionSystemsOptions::addEntry(const char* name, unsigned int color,
 void GuiCollectionSystemsOptions::createCollection(std::string inName)
 {
 	std::string name = CollectionSystemManager::get()->getValidNewCollectionName(inName);
+
+	std::string absCollectionFn = getCustomCollectionConfigPath(name);
```

Comment:
@Gemba Apologies, just back and looking at this. Shouldn't/couldn't this entire logic be reused from CollectionSystemManager somehow?

Reply:
I'm thinking just reusing the "saveCustomCollection" method, if possible.